pkgs/development/compilers/ponyc/default.nix

```
@@ -91,7 +92,12 @@ stdenv.mkDerivation (rec {
     # Replace downloads with local copies.
     substituteInPlace lib/CMakeLists.txt \
         --replace-fail "https://github.com/google/benchmark/archive/v$benchmarkRev.tar.gz" "$NIX_BUILD_TOP/deps/benchmark-$benchmarkRev.tar" \
-        --replace-fail "https://github.com/google/googletest/archive/release-$googletestRev.tar.gz" "$NIX_BUILD_TOP/deps/googletest-$googletestRev.tar"
+        --replace-fail "https://github.com/google/googletest/archive/refs/tags/v$googletestRev.tar.gz" "$NIX_BUILD_TOP/deps/googletest-$googletestRev.tar"
+
+    # The build system will run $CMAKE_CXX_COMPILER --print-file-name=libstdc++.a here, which used
+    # to work due to GCC parts leaking into Clang. This has been fixed, so we must now use gcc here.
+    substituteInPlace src/libponyc/CMakeLists.txt \
+        --replace-fail "\''${CMAKE_CXX_COMPILER}" "${gcc}/bin/gcc"
```

```suggestion
        --replace-fail "\''${CMAKE_CXX_COMPILER}" "${gcc}/bin/${stdenv.cc.targetPrefix}gcc"
```
At least this is needed to fix cross-compilation.